source/mainnet/smart-contracts/tutorials/wCCD/wCCD-interacting.rst

```diff
@@ -0,0 +1,778 @@
+.. _wCCD-interacting:
+
+========================================
+Interacting with the wCCD token protocol
+========================================
+
+.. note::
+
+    Before you start with part 2 of this tutorial, make sure you have access to a testnet node,
```

Comment:
I would do this as a bullet list:

    Before you start with part 2 of this tutorial, make sure you have:
- access to a testnet node
- the ``concordium-client`` installed
- an account created in the mobile wallet app that is funded with some CCD
- the keys from the Mobile Wallet imported into the ``concordium-client`` configuration
    If you haven't completed any of the above steps, :ref:`part 3 <piggy-bank-preparing>` of the piggy bank tutorial 
    will guide you through these setup steps. If you haven't completed any of the above steps, you can continue
    with :ref:`part 3 <wCCD-front-end-set-up>` of this tutorial. However, we recommend completing every part of this tutorial in the given order.